julia/randomMutations.jl

```diff
@@ -227,4 +227,13 @@ function deleteRandomOp(tree::Node)::Node
         end
     end
     return tree
+end
+
+# Create a random equation by appending random operators
+function genRandomTree(length::Integer)::Node
```

Comment:
genRandomTree can work using the functions defined up to this point, and is a natural extension of all other random tree functions in this file